Objects/descrobject.c

```diff
@@ -633,11 +633,40 @@ descr_reduce(PyDescrObject *descr, PyObject *Py_UNUSED(ignored))
                          PyDescr_TYPE(descr), PyDescr_NAME(descr));
 }
 
+
+static PyObject*
+classmethoddescr_reduce(PyDescrObject* descr, PyObject* Py_UNUSED(ignored))
+{
+    /* Ideally, we would want to use a different callable than eval in
+       order to get the descriptor. However, we need to ensure the pickled
+       object will not cause errors upon unpickling in older versions. */
+    PyObject* evalFunctionName = PyUnicode_FromString("eval");
+    if (evalFunctionName == NULL) {
+        return NULL;
+    }
+    PyObject* eval = _PyEval_GetBuiltin(evalFunctionName);
+    Py_DECREF(evalFunctionName);
+
+    if (eval == NULL) {
+        return NULL;
+    }
```

Comment:
Does calling `_PyEval_EvalFrameDefault` directly here work? Or maybe `PyEval_Vector`?

Reply:
Oooh sorry I got confused. In that case what you're doing is probably right.